src/ui/main.cpp

```diff
@@ -19,30 +35,10 @@
 #include <GrHttpBlocks.hpp>
 #include <GrMathBlocks.hpp>
 #include <GrTestingBlocks.hpp>
-
-#include "common/Events.hpp"
-#include "common/ImguiWrap.hpp"
-#include "common/LookAndFeel.hpp"
-#include "components/ImGuiNotify.hpp"
-
-#include "utils/EmscriptenHelper.hpp"
-
-#include <imgui_impl_opengl3.h>
-#include <imgui_impl_sdl2.h>
-#include <implot.h>
-#include <misc/cpp/imgui_stdlib.h>
-
-#include <SDL.h>
-#if defined(IMGUI_IMPL_OPENGL_ES2)
-#include <SDL_opengles2.h>
-#else
-#include <SDL_opengl.h>
-#endif
-
-#include "App.hpp"
-
-#include "common/AppDefinitions.hpp"
-#include "common/TouchHandler.hpp"
+#include <gnuradio-4.0/BlockRegistry.hpp>
```

Comment:
Some of the includes seem to be duplicates and/or unused or at least not used in the file they are included in.